[2025-05-18T02:57:51.690+0000] {logging_mixin.py:188} INFO - Changing /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T02:57:51.100329+00:00/task_id=extract_api permission to 509
[2025-05-18T02:57:51.690+0000] {logging_mixin.py:188} INFO - Failed to change /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T02:57:51.100329+00:00/task_id=extract_api permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T02:57:51.100329+00:00/task_id=extract_api'
[2025-05-18T02:57:51.717+0000] {logging_mixin.py:188} INFO - Changing /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T02:57:51.100329+00:00/task_id=extract_api permission to 509
[2025-05-18T02:57:51.717+0000] {logging_mixin.py:188} INFO - Failed to change /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T02:57:51.100329+00:00/task_id=extract_api permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T02:57:51.100329+00:00/task_id=extract_api'
[2025-05-18T02:57:51.750+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dimensional_model.extract_api manual__2025-05-18T02:57:51.100329+00:00 [queued]>
[2025-05-18T02:57:51.757+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dimensional_model.extract_api manual__2025-05-18T02:57:51.100329+00:00 [queued]>
[2025-05-18T02:57:51.757+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2025-05-18T02:57:51.768+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): extract_api> on 2025-05-18 02:57:51.100329+00:00
[2025-05-18T02:57:51.773+0000] {standard_task_runner.py:60} INFO - Started process 205 to run task
[2025-05-18T02:57:51.776+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'etl_dimensional_model', 'extract_api', 'manual__2025-05-18T02:57:51.100329+00:00', '--job-id', '2', '--raw', '--subdir', 'DAGS_FOLDER/etl_load_postgres_dag.py', '--cfg-path', '/tmp/tmpde5jdl55']
[2025-05-18T02:57:51.779+0000] {standard_task_runner.py:88} INFO - Job 2: Subtask extract_api
[2025-05-18T02:57:51.823+0000] {logging_mixin.py:188} INFO - Changing /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T02:57:51.100329+00:00/task_id=extract_api permission to 509
[2025-05-18T02:57:51.823+0000] {logging_mixin.py:188} INFO - Failed to change /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T02:57:51.100329+00:00/task_id=extract_api permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T02:57:51.100329+00:00/task_id=extract_api'
[2025-05-18T02:57:51.825+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_dimensional_model.extract_api manual__2025-05-18T02:57:51.100329+00:00 [running]> on host b360cf1752f1
[2025-05-18T02:57:52.030+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='etl_dimensional_model' AIRFLOW_CTX_TASK_ID='extract_api' AIRFLOW_CTX_EXECUTION_DATE='2025-05-18T02:57:51.100329+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-18T02:57:51.100329+00:00'
[2025-05-18T02:57:52.693+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-05-18T02:57:52.702+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl_dimensional_model, task_id=extract_api, execution_date=20250518T025751, start_date=20250518T025751, end_date=20250518T025752
[2025-05-18T02:57:52.740+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-18T02:57:52.764+0000] {taskinstance.py:3281} INFO - 1 downstream tasks scheduled from follow-on schedule check
